modules/_node-scripts/bundle/sass/.gitignore

```diff
@@ -0,0 +1 @@
+/binary
```

Comment:
The binary compiler is currently downloaded to a `binary` dir inside `liferay-portal/modules/_node-scripts/bundle/sass`.